src/js/components/AdminPanels.tsx

```diff
@@ -173,7 +198,9 @@ export function DropDownMenuContainer({
   const [redisChanged, updateRedisStatus] = useRedisStatus()
   const [thisMenu, setThisMenu] = useState(menu)
 
-  const handleSelect = (item: MenuItem) => {
+  useEffect(() => setThisMenu(menu), [menu])
```

Comment:
Not for now, but I recently gave a closer look to https://react.dev/learn/you-might-not-need-an-effect which made me realize about some alternatives / better use for `useEffect`. Specifically, for defining states from props there are a few suggestion which might be better.